src/main/java/org/jabref/gui/preferences/general/GeneralTab.java

```diff
@@ -127,10 +136,18 @@ public void initialize() {
         backupDirectory.textProperty().bindBidirectional(viewModel.backupDirectoryProperty());
         backupDirectory.disableProperty().bind(viewModel.createBackupProperty().not());
 
+        ActionFactory actionFactory1 = new ActionFactory(Globals.getKeyPrefs());
```

Comment:
You can remove this, as there is already an instance of ActionFactory

Reply:
OK！:)